### ISSUE TITLE
DJV 2.0 Feature request: Add Center Marker (Center Cross)

### ISSUE THREAD

**Issue by aberkl** [darbyjohnston/DJV#322]
Sometimes it is very handy to have a center marker:

![CenterMarker](https://user-images.githubusercontent.com/9915206/86471199-3cdd1d00-bd3d-11ea-84f8-ae383320f3ff.jpg)

Note: In order to perfectly match the image center it might be necessary to adapt the width of the marker-lines to the image dimensions. 

For both H and V of the image being odd numbers the width should also be odd, for H and V being even the width should be even as well. Consequently this may result in horizontal lines being odd and vertical lines being even or vice versa.

**Comment by darbyjohnston** [comment]
Would the center marker respond to the image zoom or maintain a fixed size?

**Comment by aberkl** [comment]
> Would the center marker respond to the image zoom or maintain a fixed size?

I guess not to confuse the user it should respond to zoom.

**Comment by darbyjohnston** [comment]
Actually I think it would be confusing having it zoom, it seems more like a heads-up display element like the grid and image border. Plus if it zooms at some point it becomes unusable when zooming in/out too far. Zoom in too far and it could cover pixels you are trying to see, zoom out too far and it becomes invisible.

**Comment by aberkl** [comment]
> [...] it seems more like a heads-up display element like the grid and image border. Zoom in too far and it could cover pixels you are trying to see, zoom out too far and it becomes invisible.

Good point. Make it part of HUD which can be toggled using "CTRL + U"? That should solve the issue.

**Comment by darbyjohnston** [comment]
Sorry for the long delay, I finally added a center marker; here's a screenshot:

<img width="1924" height="1140" alt="Image" src="https://github.com/user-attachments/assets/f60c14b6-4261-43a3-b806-d847cb0f87a6" />

Hopefully the rounding functions give similar behavior as you described.

**Comment by aberkl** [comment]
Center marker now available in DJV 3.
